archetypes/helidon/src/main/archetype/se/custom/observability.xml

```diff
@@ -29,17 +35,8 @@ tracing:
     host: "192.168.1.3"   # JAEGER_ENDPOINT
     port: 14240           # JAEGER_ENDPOINT
     path: "/api/traces/mine"   # JAEGER_ENDPOINT
-    token: "token"        # JAEGER_AUTH_TOKEN
-    # Either token or username/password
-    #username:  "user"     # JAEGER_USER
-    #password: "pass"      # JAEGER_PASSWORD
-    propagation: "jaeger" # JAEGER_PROPAGATION either "jaeger" or "b3"
-    log-spans: false      # JAEGER_REPORTER_LOG_SPANS
-    max-queue-size: 42    # JAEGER_REPORTER_MAX_QUEUE_SIZE
-    flush-interval-ms: 10001 # JAEGER_REPORTER_FLUSH_INTERVAL
-    sampler-type: "remote"# JAEGER_SAMPLER_TYPE (https://www.jaegertracing.io/docs/latest/sampling/#client-sampling-configuration)
+    sampler-type: "ratio" # JAEGER_SAMPLER_TYPE (https://www.jaegertracing.io/docs/latest/sampling/#client-sampling-configuration)
```

Comment:
I am not really sure what we want to accomplish here. If this is to make it easy for users to set up Jager, then we should ONLY have `host` and `port` (and maybe `sampler-type` and `sampler-param`) defined,. and the host should be `localhost` - as this will work out of the box with local Jaeger in docker.
The port should be set to `14250` (default for Jaeger)
Everything else is kind of confusing for first time user.